.github/workflows/python-package.yaml

```diff
@@ -0,0 +1,30 @@
+name: Python package
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Not related to this PR, but wanted to comment that this line is not in bioutils repo

Reply:
I may have already said this somewhere else (thought i had) but in general, everything is pretty out of sync w/ the template. Not sure if that's something we want to tackle iteratively or do a single "clean sweep"/reset at some point.